src/components/Modal.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const Modal = ({title, children, onExitClick}) => {
+  return (
+    <div>
+      <div class="modal is-active">
+        <div onClick={onExitClick} class="modal-background"></div>
+        <div class="modal-card">
+          <header class="modal-card-head">
+            <p class="modal-card-title">{title}</p>
```

Comment:
Like discussed, we may reevaluate later whether this needs to be here, or if we strictly need some with and some without titles, add a different modal component as well